cmds/tui.c

```diff
@@ -192,7 +192,7 @@ enum tui_mode {
 };
 
 static char *report_sort_key[] = { OPT_SORT_KEYS, "total_avg", "total_min", "total_max", "self",
-				   "self_avg",	  "self_min",  "self_max",  "call" };
+				   "self_avg",	  "self_min",  "self_max",  "call",	 "size" };
```

Comment:
Please add a comma after the "size" so that it can be aligned properly.

Reply:
I updated it.